test/utils/toMomentObject_spec.js

```diff
@@ -20,6 +20,10 @@ describe('toMomentObject', () => {
     expect(toMomentObject()).to.equal(null);
   });
 
+  it('output has time of 12PM', () => {
```

Comment:
This becomes a bit harder to test I think because it... depends on a lot of different factors. I mean, I guess we can check that the first of april/last of march and first of march/last of april do not make isSameDay true but that seems like a bad test?